### ISSUE TITLE
Service letter accessibility issue – title is not set up

### ISSUE THREAD

**Issue by Igor-Ino** [LibraryOfCongress/concordia#1540]
**What behavior did you observe? Please describe the bug**
A clear and concise description of what you experienced.

**How can we reproduce the bug?**

1. Open Service Letter in Adobe Acrobat Pro DC
2. In left top menu go to File>Document properties
3. Click on tab "Initial View"
4. See Show: File Name, click on dropdown menu and choose Document title
**What is the expected behavior?**
A clear and concise description of what you expected to happen.Document tile should be Initial View
**Got screenshots? This helps us identify the issue**
Add screenshots to help explain your problem.

**Desktop (please complete the following information):**

-   OS: all OSes and browsers
![Initial View](https://user-images.githubusercontent.com/80710853/133688349-243d813c-3d1b-4333-8d93-3c6edc926041.jpg)

**Comment by jkueloc** [comment]
MOved to Jira: [CONCD-17](https://staff.loc.gov/tasks/browse/CONCD-17)

**Comment by jkueloc** [comment]
This issue is closed as the work has been moved to the Library's Jira repo as noted above.
